utils/UnicodeData/confusables.txt

```diff
@@ -1,11 +1,11 @@
 ﻿# confusables.txt
-# Date: 2016-06-16, 13:41:30 GMT
-# © 2016 Unicode®, Inc.
+# Date: 2020-02-13, 01:38:49 GMT
+# © 2020 Unicode®, Inc.
 # Unicode and the Unicode Logo are registered trademarks of Unicode, Inc. in the U.S. and other countries.
 # For terms of use, see http://www.unicode.org/terms_of_use.html
 #
-# Unicode Security Mechanisms for UTR #39
-# Version: 9.0.0
+# Unicode Security Mechanisms for UTS #39
+# Version: 13.0.0
```

Comment:
I will confirm if we can simply update this file in our repo.

Reply:
Okay!